### ISSUE TITLE
Changing the form fields depending on issue type is not working

### ISSUE THREAD

**Issue by pclais** [getsentry/sentry-jira#43]
When changing issue type in Sentry, the form is not changing depending on the required fields. Thus, we cannot create a JIRA task.
Thanks;
Paul

**Comment by phun-ky** [comment]
I experience the same issue, made a new issue: https://github.com/thurloat/sentry-jira/issues/75
